app/_locales/en/messages.json

```diff
@@ -18,13 +18,13 @@
     "message": "After you’ve signed with your wallet, click on 'Get Signature' to receive the signature"
   },
   "QRHardwareSignRequestGetSignature": {
-    "message": "Get Signature"
+    "message": "Get signature"
```

Comment:
@adonesky1 et @Gudahtt I apologize for my ignorance but could you school me on how the new transaltion system works? Will these changes be picked up and retroactively applied to all languages?

Reply:
Oh that's a great question! I was fully prepared to create more PRs to cover all the other locales. If there is an automated process or some way to delegate that task it would make my day.

Reply:
Just posting @rickycodes [slack DM](https://consensys.slack.com/archives/C03SV1QLE7N/p1659971535884859) here for reference
> Will these changes be picked up and retroactively applied to all languages?
yes! once the change is merged in, Blend will be notified via the crowdin integration and a follow up PR should be opened by crowdin once their aforementioned updates are ready

Reply:
@brad-decker can you explain what you mean by "retroactively applied to all languages"?